arches/app/models/fields/i18n.py

```diff
@@ -223,9 +222,11 @@ def get_prep_value(self, value):
         return I18n_String(value, attname=self.attname, use_nulls=self.use_nulls)
 
     def get_db_prep_save(self, value, connection):
-        """Override to avoid the optimization from Django 4.2 that
-        immediately returns `value` if it is None."""
```

Comment:
Instead of special-casing None here, I fixed it closer to the source where you see fallbacks to things like `"'{}'::jsonb"` where the python value None is not appropriate for the SQL.